lib/httpRequestBuilder.js

```diff
@@ -60,7 +60,8 @@ function requestBuilder (defaults) {
     }
 
     if (bodyBuf && bodyBuf.length > 0) {
-      headers['Content-Length'] = '' + bodyBuf.length
+      const uuidCount = (bodyBuf.toString().match(/\[<uuid>\]/g) || []).length
+      headers['Content-Length'] = `${bodyBuf.length + (uuidCount * 28)}`
```

Comment:
The maths relating to this content-length is wrong, I think. ```content-length = bodyBuf.length - ('<uuid>'.length* uuidCount) + (uuidCount * 28)```

Because the bodyBuf string will remove the '<uuid>' tokens to replace them with the uuid's

Also is an uuid not 16 bytes in length?

Thanks!